tests/experimental/post_training/test_low_opset_model.py

```diff
@@ -0,0 +1,128 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+# pylint: disable=redefined-outer-name
+
+import pytest
+
+import os
+
+import torch
+from torchvision import models
+import onnx
+
+from nncf.experimental.onnx.datasets.imagenet_dataset import infer_input_shape
+from tests.common.helpers import TEST_ROOT
+
+
+MODEL_NAMES = [
+    'resnet18',
+    'mobilenet_v2',
+    'inception_v3',
+    'googlenet',
+    'vgg16',
+    'densenet121'
+]
+
+MODELS = [
+    models.resnet18(),
```

Comment:
I suggest to test the models that are known to failed without this PR. Do they?

Reply:
I added models in tests/onnx/quantization/test_classification_models_graph.py.
What additional models should be tested?